internal/pkg/cli/job_deploy.go

```diff
@@ -80,6 +95,34 @@ func (o *deployJobOpts) RecommendedActions() []string {
 	return nil
 }
 
+func (o *deployJobOpts) validateJobName() error {
+	names, err := o.ws.JobNames()
+	if err != nil {
+		return fmt.Errorf("list jobs in the workspace: %w", err)
+	}
+	for _, name := range names {
+		if o.Name == name {
+			return nil
+		}
+	}
+	return fmt.Errorf("job %s not found in the workspace", color.HighlightUserInput(o.Name))
+}
+
+func (o *deployJobOpts) validateEnvName() error {
+	if _, err := o.targetEnv(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *deployJobOpts) targetEnv() (*config.Environment, error) {
```

Comment:
We probably don't need this method since
``` go
func (o *deployJobOpts) validateEnvName() error {
	_, err := o.store.GetEnvironment(o.AppName(), o.EnvName)
	if err != nil {
		return fmt.Errorf("get environment %s configuration: %w", o.EnvName, err)
	}
	return nil
}
```
should be good enough

Reply:
Okay, thanks!